route/build/src/histVars_data.f90

```diff
@@ -342,6 +369,8 @@ SUBROUTINE read_restart(this, restart_name, ierr, message)
       integer(i4b)                       :: ixWaterH              ! temporal water height variable index
       integer(i4b)                       :: ixFloodV              ! temporal flood volume variable index
       integer(i4b)                       :: ixInflow              ! temporal inflow variable index
+      integer(i4b)                       :: ixTracerMass          ! temporal tracer mass variable index
```

Comment:
As above link the meaning of tracer to solute in the comment.

Reply:
changed `ixTracerMass` to `ixSoluteMass`